README.md

```diff
@@ -18,64 +17,75 @@ Then, add one or more of the modules below:
+file. Can be used for detecting accidental changes of events.
+
+Installation:
 
-To use, just annotate a base trait with `@org.virtuslab.akkasaferserializer.SerializabilityTrait`:
+Add to `plugins.sbt`:
 
 ```scala
-@SerializabilityTrait
-trait MySerializable
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" % "sbt-dump-event-schema" % commit
 ```
 
-Installation:
+To enable the plugin for a specific project, use:
 
 ```scala
-libraryDependencies += repo %% "akka-serializability-checker-library" % commit
-libraryDependencies += compilerPlugin(repo %% "akka-serializability-checker-plugin" % commit)
+enablePlugins(DumpEventSchemaPlugin)
 ```
 
-### 3. Dump Schema
+You also have to change one setting responsible for resolving companion compiler plugin:
 
-An sbt plugin that allows for dumping schema of events to a file. Can be used for detecting accidental changes of
-events.
+```scala
+dumpEventSchema / dumpEventSchemaCompilerPlugin := "com.github.VirtusLab.akka-serialization-helper" % "dump-event-schema-compiler-plugin" % commit 
+```
 
-Installation:
+### 3. Serializer
+
+Simple [Borer](https://github.com/sirthias/borer) based Akka serializer. It uses codecs, provided by Borer, that are
+generated during compile time (so serializer won't crash during runtime like reflection-based serializers may do).
 
-Add to `plugins.sbt`:
 ```scala
-libraryDependencies += repo % "sbt-dumpschema" % commit
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "borer-akka-serializer" % commit
 ```
-To enable the plugin for a specific project, use:
+
+It may also be worth including additional codecs for common types that are missing in Borer standard library:
+
 ```scala
-enablePlugins(DumpSchemaPlugin)
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "borer-extra-codecs" % commit
 ```
 
 ## Comparison with other Akka Serializers
 
 | Serializer | Jackson | Circe | Protobuf v3 | Avro | Borer |
 |:---|:---|:---|:---|:---|:---|
-| Data formats | JSON or CBOR | JSON | binary or JSON | binary or JSON | JSON or CBOR |
+| Data formats | JSON or CBOR | JSON | JSON or binary | JSON or binary | JSON or CBOR |
 | Scala support | with [`jackson-module-scala`](https://github.com/FasterXML/jackson-module-scala) <ul><li>lacks support of basic scala types like `Unit`</li><li>without explicit annotation doesn't work with generics extending `AnyVal`</ul> | perfect | with [`ScalaPB`](https://scalapb.github.io) generates Scala ADTs based on protobuf definitions | with [`Avro4s`](https://github.com/sksamuel/avro4s) library generaters Avro schema based on Scala ADTs | perfect
-| Akka support | out of the box | requires custom serializer | requires custom serializer | requires custom serializer | out of the box |
+| Akka support | out of the box | requires custom serializer | requires custom serializer | requires custom serializer | requires custom serializer |
```

Comment:
Wow really other than Jackson, none of them ships with a ready Akka serializer? 😯 (Kryo does for sure tho)

Reply:
As in, maybe not _ships with_ but _has a ready library somewhere out there on github that provides_